src/main/java/com/google/codeu/servlets/ReviewServlet.java

```diff
@@ -0,0 +1,53 @@
+package com.google.codeu.servlets;
+
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.Location;
+
+import com.google.appengine.api.datastore.*;
```

Comment:
Same thing as with Datastore.java. Best if each specific package type is imported for minimum ambiguity.